week2/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang = "en-us">
+    <head>
+        <meta charset="uft-8">
+        <title>My first HTML</title>
+    </head>
+    <body>
+        <header>
+            <nav>
+                <a href="pages/about.html">About</a>
+            </nav>
+            <h1>
+                My First HTML Page
+            </h1>
+        </header>
+        
+        <main>
+            <article>
+                <h2>
+                    <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element" title="HTML Elements" target="_blank">HTML Elements</a>
+                </h2>
+                <a href="https://commons.wikimedia.org/wiki/File:HTML5_logo_and_wordmark.svg">
+                    <img
+                    src="images/HTML5_logo_and_wordmark.svg.png"
+                    alt="HTML5 Logo"
+                    width="200"
+                    height="200"/>
+                </a>
+                <p>
+                    HTML elements <em>usually</em> consist of opening and closing tags and content.
+                </p>
+                <h3>
+                    Empty Elements
+                </h3>
+                <p>
+                    Some HTML elements don't have closing tags and content such as &#60img&#62
+                </p>
+            </article>
+            <aside>
```

Comment:
&lt;aside&gt; tag is frequently presented as sidebars. This is the main content, so it should be inside article or section tag.